browser_history/__init__.py

```diff
@@ -8,18 +8,44 @@ def get_history():
     browsers for the system platform.
 
     :return: Object of class :py:class:`browser_history.generic.Outputs` with the
-        data member entries set to list(tuple(:py:class:`datetime.datetime`, str))
+        data member histories set to list(tuple(:py:class:`datetime.datetime`, str))
 
     :rtype: :py:class:`browser_history.generic.Outputs`
     """
-    output_object = generic.Outputs()
+    output_object = generic.Outputs(fetch_type="history")
     subclasses = generic.Browser.__subclasses__()
     for browser_class in subclasses:
         try:
             browser_object = browser_class()
-            browser_output_object = browser_object.fetch()
-            output_object.entries.extend(browser_output_object.get())
+            browser_output_object = browser_object.fetch_history()
+            output_object.histories.extend(browser_output_object.histories)
         except AssertionError:
             utils.logger.info("%s browser is not supported", browser_class.name)
-    output_object.entries.sort()
+    output_object.histories.sort()
+    return output_object
+
+
+def get_bookmarks():
+    """This method is used to obtain browser bookmarks of all available and supported
+    browsers for the system platform.
+
+    :return: Object of class :py:class:`browser_history.generic.Outputs` with the
+        data member bookmarks set to
+        list(tuple(:py:class:`datetime.datetime`, str, str, str))
+
+    :rtype: :py:class:`browser_history.generic.Outputs`
+    """
+    output_object = generic.Outputs(fetch_type="bookmarks")
+    subclasses = generic.Browser.__subclasses__()
+    for browser_class in subclasses:
+        try:
+            browser_object = browser_class()
+            assert browser_object.bookmarks_file is not None
+            browser_output_object = browser_object.fetch_bookmarks()
+            output_object.bookmarks.extend(browser_output_object.bookmarks)
+        except AssertionError:
+            utils.logger.info(
+                "%s browser is not supported for bookmarks", browser_class.name
+            )
```

Comment:
I noticed the log messages were a bit misleading (It said `Safari browser is not supported for bookmarks` when in fact it was supported for bookmarks but wasn't supported on Linux - which is what the assertion error returned).
```suggestion
        try:
            browser_object = browser_class()
            assert browser_object.bookmarks_file is not None, f"Bookmarks are not supported on {browser_class.name}"
            browser_output_object = browser_object.fetch_bookmarks()
            output_object.bookmarks.extend(browser_output_object.bookmarks)
        except AssertionError as e:
            utils.logger.info("%s", e)
```

Reply:
Its not supported for bookmarks also right, as of now that is unless someone adds a bookmark parser for it

Reply:
but I get what you mean